starport/cmd/cmd.go

```diff
@@ -89,6 +89,15 @@ func printEvents(bus events.Bus, s *clispinner.Spinner) {
 	}
 }
 
+func flagSetPath(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringP(flagPath, "p", ".", "sets the working directory for the command")
```

Comment:
I believe we should keep command specific descriptions instead, this one is too generic and may not make sense for all users.